.github/workflows/black.yml

```diff
@@ -0,0 +1,17 @@
+name: black
+
+on:
```

Comment:
For the `tacd`we have `on: pull_request` **and** `on: push` in our [workflow `.yaml`s](https://github.com/linux-automation/tacd/blob/main/.github/workflows/tacd-daemon.yaml). I think that should be the same here?

Reply:
LGTM. I've changed this here.